draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -563,14 +564,14 @@ but splitting cannot be supported.
 Path validation
 induces a delay when switching as packets are buffered. Further, congestion control state is
 reset on a new path and needs to ramp up. When frequent handovers happen, 
-splitting traffic over multiple paths simulaneously can be beneficial for
+splitting traffic over two paths simulaneously can be beneficial for
```

Comment:
I haven't read the full document in a while. If you feel this is editoral, then we should go ahead and change that. If it's a design choice (i.e. restricting to two paths instead of more), then it might be good to add a sentence stating the presented approach could be extended to more than two paths.

Reply:
I agree with Maxime - ATSSS as practiced in Rel 16, and apparently Rel 17, is closely tied to current UE capabilities, so, two paths. ISTM that we would want to point out that more than two paths could pop up at any time, and ISTM unlikely that the QUIC working group would be interested in preserving the two-path limitation in work that they do.